lib/attributor/attribute.rb

```diff
@@ -137,8 +137,7 @@ def example_from_options(parent, context)
       when ::Regexp
         val.gen
       when ::Array
-        # TODO: handle arrays of non native types, i.e. arrays of regexps.... ?
-        val.pick
+        val
```

Comment:
So this is most definitely a breaking change for anything that'd be using an array. 

Hopefully we don't have many (or any?) cases of that.